README.md

```diff
@@ -1,2 +1,17 @@
 # deepcrawl
 A DeepCrawl API wrapper for Node
+
+```javascript
+var DeepCrawl = require('./');
+
+var deepCrawl = new DeepCrawl({
+  apiKey: '[your api key]',
+  apiId: '[your api id]',
```

Comment:
Oh, I just realized this is the README. I thought this was a config file for some reason